src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const KEYS = Object.values(robot);
+  const VALUES = Object.keys(robot);
+  const INVERSED_ROBOT = {};
+  const UNIQUE_KEYS = [ ...new Set(KEYS) ];
+
+  if (UNIQUE_KEYS.length !== VALUES.length) {
+    return null;
+  }
+
+  for (let i = 0; i < KEYS.length; i++) {
```

Comment:
The for loop is correct, but it's not efficient. You are iterating over the keys and values separately, which requires two iterations. Instead, you could use the 'entries' method of the Object object to iterate over the keys and values at the same time.